vsintegration/src/FSharp.Editor/Classification/ClassificationService.fs

```diff
@@ -20,7 +22,42 @@ open Microsoft.CodeAnalysis.ExternalAccess.FSharp.Classification
 // IVT, we'll maintain the status quo.
 #nowarn "44"
 
+#nowarn "57"
+
+open FSharp.Compiler.Range
 open FSharp.Compiler.SourceCodeServices
+open FSharp.Compiler.SourceCodeServices.Lexer
+
+type SemanticClassificationData = (struct(FSharp.Compiler.Range.range * SemanticClassificationType)[])
+type SemanticClassificationLookup = IReadOnlyDictionary<int, ResizeArray<struct(FSharp.Compiler.Range.range * SemanticClassificationType)>>
+
+[<Sealed>]
+type DocumentCache<'Value when 'Value : not struct>() =
+    let cache = new MemoryCache("fsharp-cache")
+    let policy = CacheItemPolicy(SlidingExpiration = TimeSpan.FromSeconds 2.)
+
+    member _.TryGetValueAsync(doc: Document) = async {
+        let! ct = Async.CancellationToken
+        let! currentVersion = doc.GetTextVersionAsync ct |> Async.AwaitTask
+
+        match cache.Get(doc.Id.ToString()) with
+        | null -> return ValueNone
+        | :? (Microsoft.CodeAnalysis.VersionStamp * 'Value) as value ->
```

Comment:
You can drop the `Microsoft.CodeAnalysis.`

Reply:
Sure